python_src/src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -141,7 +144,7 @@ def process_files(self) -> Iterable[pyarrow.table]:
         yield_threshold = max(10, max_workers * 3)
 
         process_logger = ProcessLogger(
-            "create_parquet_table",
+            "create_pyarrow_tables",
```

Comment:
renamed this to make it clearer in the logs. we also have "parquet table creator" process that its easy to get confused with.